charts/Makefile

```diff
@@ -25,19 +25,14 @@ vault:
 	@echo "Installing vault ..."
 	kubectl create namespace m4d-system || true
 	helm repo add hashicorp https://helm.releases.hashicorp.com
-	helm install vault hashicorp/vault --version 0.9.1 \
-                --set "global.openshift=$(WITH_OPENSHIFT)" \
-                --set injector.enabled=false \
-                --set "server.dev.enabled=true" \
-		--values ../third_party/vault/plugin-secrets-kubernetes-reader/values.yaml \
+	helm dependency update vault
+	helm install vault vault \
+                --set "vault.global.openshift=$(WITH_OPENSHIFT)" \
```

Comment:
I think that global is special and is shared with all subcharts so it should be set at top level, are we sure this one works?

Reply:
I checked that and it worked ok,.. for the testing I had to enable the injector to validate that the resources are generated as expected as openshift flag affects the injector deployment.